eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.9.0.0
 
+* Prevent `DREP` expiry when there are no active governance proposals to vote on. #3729
+  * Add `updateNumDormantEpochs` function in `ConwayEPOCH` to update the dormant-epochs counter
+  * Add `VState` to the State for `ConwayGOV` by introducing a new `GovRuleState` type, and update `ConwayLEDGER` accordingly.
+  * Add a call to a new `updateDRepExpiry` function in the existing `addAction` function in `ConwayGOV`, that updates `DRep` expiries and resets the dormant-epochs counter to zero as soon as a new governance proposal is successfully submitted to vote on, after a contiguous set of dormant-epochs.
```

Comment:
Not sure if this last entry  has to be in the CHANGELOG, since it doesn't affect the "surface"

Reply:
Hmm, I wonder myself :thinking:

Reply:
No we don't need to add it. `addAction` is not even an exported function.

I understand that it chagnes the semantics of the RULE, but if we were to add every semantic change to every rule we make changelog would be huge :grin: 

We only need to add things into the changelog that would be useful to the downstream users, i.e. the "surface" people :smile:

Reply:
Noted! :memo:

Thanks @teodanciu and @lehins :raised_hands: